__tests__/app.test.js

```diff
@@ -71,4 +71,18 @@ describe("GET: /api/articles/:article_id", () => {
             expect(response.body.msg).toBe('Invalid ID');
         })
     })
+})
+
+describe('/api/articles', () => {
+    test('GET:200 sends an array of article objects with the correct keys', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+          expect(response.body.articles).toEqual(expect.any(Array));
+          expect(Object.keys(response.body.articles[0])).toEqual(
```

Comment:
An extra assertion here to check how many objects are in the array (length) may be useful here?